src/contract/force-move-app.ts

```diff
@@ -0,0 +1,30 @@
+import {getVariablePart} from '../contract/state';
+import {ethers} from 'ethers';
+import ForceMoveAppArtifact from '../../build/contracts/ForceMoveApp.json';
+import {State} from './state';
+
+const ForceMoveAppContractInterface = new ethers.utils.Interface(ForceMoveAppArtifact.abi);
+
+export async function validTransition(
+  fromState: State,
+  toState: State,
+  forceMoveAppAddress: string,
+  signer: ethers.Signer,
+): Promise<boolean> {
+  const numberOfParticipants = toState.channel.participants.length;
+  const fromVariablePart = getVariablePart(fromState);
+  const toVariablePart = getVariablePart(toState);
+  const turnNumB = 0; // This isn't actually used by the contract so any value works
```

Comment:
Are you sure this comment is correct? I think there isn't *yet* a ForceMoveApp that uses `turnNumB` but I expect there to be shortly. For example, a payments game where the rules depend on whose move it is.

Reply:
Ah, this was copied from the `ConsensusApp` where it's not used.  Will update.